CONTRIBUTING.md

```diff
@@ -11,26 +28,118 @@
   - [Slack](#slack)
 <!-- tocstop -->
 
-# Contributing
+## Welcome 👋
 
 Contributions are always welcome. Before contributing please read the
-[code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md) and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `main`. To contribute,
-[fork](https://help.github.com/articles/fork-a-repo/) this repository, commit your changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
-
-Note that our [code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md) applies to all platforms and venues related to this project; please follow it in all your interactions with the project and its participants.
+[code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md)
+and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `main`. To contribute,
+[fork](https://help.github.com/articles/fork-a-repo/) this repository, commit your
+changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
 
+Note that our [code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md)
+applies to all platforms and venues related to this project; please follow it in all
+your interactions with the project and its participants.
 
 ## Observability Pack Guidelines
 
-We encourage all contributors to actively engage in the creation and maintenance of Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+We encourage all contributors to actively engage in the creation and maintenance of
+Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+
+- `Step 1`: Review the [pack template config](./_template/config.yml) file for a definition of how to create a pack.
+- `Step 2`: Review the [Pack Config File Schema](https://github.com/newrelic/newrelic-observability-packs/blob/f3acfd3efc34899c088c01b48c2e2f2210dd2657/docs/config-spec.md) for structure and limits you need to consider.
```

Comment:
This link should be to the main branch rather than a commit (which will end up out of date)
```suggestion
- `Step 2`: Review the [Pack Config File Schema](https://github.com/newrelic/newrelic-observability-packs/blob/main/docs/config-spec.md) for structure and limits you need to consider.
```